index.js

```diff
@@ -78,18 +78,22 @@ class BitbucketScm extends Scm {
      */
     _parseUrl(config) {
         const repoInfo = getRepoInfo(config.checkoutUrl);
-        const branchUrl = `${REPO_URL}/${repoInfo.username}/${repoInfo.repo}` +
-            `/refs/branches/${repoInfo.branch}?access_key=${config.token}`;
+        const branchUrl =
+            `${REPO_URL}/${repoInfo.username}/${repoInfo.repo}/refs/branches/${repoInfo.branch}`;
         const options = {
             url: branchUrl,
             method: 'GET',
-            json: true
+            json: true,
+            auth: {
+                bearer: config.token
+            }
         };
 
         return this.breaker.runCommand(options)
             .then((response) => {
                 if (response.statusCode !== 200) {
-                    throw new Error(`STATUS CODE ${response.statusCode}: ${response.body}`);
+                    throw new Error(
```

Comment:
Right now it will give:
```
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "STATUS CODE 401: {\"error\":{\"message\":\"Access token expired. Use your refresh token to obtain a new access token.\"}}"
}
```